int32 id
geometry_msgs/Pose center
geometry_msgs/PoseStamped[] grasping_points
